config/ms.yml

```diff
@@ -13,20 +14,33 @@ example_terms:
 - MS_0000000
 
 entries:
-- prefix: /releases/
```

Comment:
PURLs are supposed to be permanent. You can't just drop all these `/releases/` PURLs.

Reply:
It's safer to put all `prefix` entries at the bottom, after all the `exact` entries.